frontend/src/Components/Modal.js

```diff
@@ -15,7 +16,8 @@ export default function Modal() {
         setModal(!modal)
     }
     const [inputs, setInputs] = useState({});
-    const [type, setType] = useState("");
+    const [type, setType] = useState("keras");
+    const notifySuccess = () => toast("Model uploaded succesfully!")
```

Comment:
This line im not sure is used, se debe remover for purposes of cleaning the code pero no es un stopper.